nautobot/docs/development/core/local-k8s.md

```diff
@@ -0,0 +1,126 @@
+# Using Local Kubernetes Cluster
+
+This guide will help you set up a local Kubernetes cluster using [Kind](https://kind.sigs.k8s.io/) and connect running Nautobot containers to the Kind network.
+
+## Kind Installation
+
+To install Kind on macOS:
+
+```bash
+brew install kind
+```
+
+To install Kind on Linux:
+
+```bash
+export KIND_VERSION=0.24.0
+curl -fsSL \
+    "https://kind.sigs.k8s.io/dl/v$KIND_VERSION/kind-linux-amd64" \
+    -o /usr/local/bin/kind
+chmod +x /usr/local/bin/kind
+```
+
+To create a new Kind cluster:
```

Comment:
Can we add some docs on how to clean up when done? Also clarify which of the below steps need to be repeated later and which are one-off tasks.

Reply:
Added